en/flight_modes/land.md

```diff
@@ -26,30 +26,132 @@ Landing is affected by the following parameters:
+
+1. **Fly to landing location**: The aircraft flies at the current altitude to the landing descent orbit.
+2. **Descend to approach altitude**: On reaching the orbit, the vehicle will descend to the approach altitude.
+   Once reaching the approach altitude, the airframe will continue on the orbit until reaching the final approach exit (tangential exit towards landing waypoint).
+4. **Final approach**: The aircraft will follow the landing approach slope until the flare altitude is reached.
+5. **Flare**: The vehicle flares until touching down.
+
+![Fixed-wing landing](../../assets/flying/fixed-wing_landing.png)
+
+### Landing Approach
+
+The vehicle tracks the landing slope (generally at a slower speed than cruise) until reaching the flare altitude.
+
+Parameter | Description
+--- | ---
+[FW_LND_ANG](../advanced_config/parameter_reference.md#FW_LND_ANG) | The maximum achievable landing approach slope angle. Note that smaller angles may still be commanded via the landing pattern mission item.
+[FW_LND_EARLYCFG](../advanced_config/parameter_reference.md#FW_LND_EARLYCFG) | Optionally deploy landing configuration during the landing descent orbit (e.g. flaps, spoilers, landing airspeed).
+[FW_LND_AIRSPD](../advanced_config/parameter_reference.md#FW_LND_AIRSPD) | Calibrated airspeed setpoint during landing.
+[FW_FLAPS_LND_SCL](../advanced_config/parameter_reference.md#FW_FLAPS_LND_SCL) | Flaps setting during landing.
+[FW_LND_THRTC_SC](../advanced_config/parameter_reference.md#FW_LND_THRTC_SC) | Altitude time constant factor for landing (overrides default [TECS tuning](../config_fw/advanced_tuning_guide_fixedwing.md)).
+
+### Flaring / Roll-out
+
+Flaring consists of a switch from altitude tracking to a shallow sink rate setpoint and constraints on the commandable throttle, resulting in nose up manuevering to slow the descent and produce a softer touchdown.
+
+The flaring altitude is calculated during the final approach via "time-to-impact" ([FW_LND_FL_TIME](#FW_LND_FL_TIME)) and the approach descent rate.
+An additional safety parameter [FW_LND_FLALT](#FW_LND_FLALT) sets the minimum altitude at which the vehicle will flare (if the time based altitude is too low to allow a safe flare maneuver).
+
+If belly landing, the vehicle will continue in the flaring state until touchdown, land detection, and subsequent disarm. For runway landings, [FW_LND_TD_TIME](#FW_LND_TD_TIME) enables setting the time post flare start to pitch down the nose (e.g. consider tricycle gear) onto the runway ([RWTO_PSP](#RWTO_PSP)) and avoid bouncing. This time roughly corresponds to the touchdown post flare, and should be tuned for a given airframe during test flights only after the flare has been tuned.
+
+Parameter | Description
+--- | ---
+<a id="FW_LND_FL_TIME"></a>[FW_LND_FL_TIME](../advanced_config/parameter_reference.md#FW_LND_FL_TIME) | Time before impact (at current descent rate) at which the vehicle should flare. 

```

Comment:
FYI, note I added anchors in all these. This pattern allows us to link from anywhere in the page to the table (maintaining context of the page) rather than linking to the parameter page directly.